interpreter/terminal_interface/start_terminal_interface.py

```diff
@@ -283,6 +283,23 @@ def start_terminal_interface(interpreter):
         # Doesn't matter
         pass
 
+    # At some point in the future these model names redirects wont be necessary anymore, but legacy names will remain for a while
+    if interpreter.model == "gpt-4" or interpreter.model == "gpt-4-32k":
+        interpreter.model = "gpt-4-1106-preview"
```

Comment:
(btw going to remove these lines after merging, as I've found `gpt-4-1106-preview` to be worse in some ways and we should let folks choose!)